Eastern-Research-Group/expert-query
# Feature/93 endpoints manually run etl

## Related Issues:
* [EQ-93](https://jira.epa.gov/browse/EQ-93)

## Main Changes:
* Removed some code for testing internal routes. 
  * I've been commiting code directly to the develop branch, in order to test ways that we can manually run the etl process from cloud.gov without exposing the methods to a public api. 
* Added tasks that can be run from the cloud.gov cf cli.  On the most basic level, tasks are just scripts in our package.json file. 
  * To run locally: `npm run <SCRIPT NAME>`
    * Ex: `npm run etl_glossary`
  * To run on cloud.gov: `cf run-task <APP NAME> --command "npm run <SCRIPT NAME>" --name <TASK ALIAS>`
    * Ex: `cf run-task expert-query-etl-dev --command "npm run etl_glossary" --name etl_glossary`
    * The `--name <TASK ALIAS>` param is optional. This is just makes it easier to check the status of the task.
  * Related Documentation: 
    * https://docs.cloudfoundry.org/devguide/using-tasks.html
    * https://cli.cloudfoundry.org/en-US/v6/run-task.html 

## Steps To Test:
1. Run the following commands from the `etl` folder and verify they work
    * `npm run etl_all` - Feel free to cancel this early if you don't want to wait for all of the data to be loaded
    * `npm run etl_database`  - Feel free to cancel this early if you don't want to wait for all of the data to be loaded
    * `npm run etl_domain_values`
    * `npm run etl_glossary`
2. Run `npm run start` from the `etl` folder and verify it works.
    * Feel free to cancel this early if you don't want to wait for all of the data to be loaded
3. After this gets merged in, rerun the above tests from cloud.gov.
